src/Keboola/StorageApi/Client.php

```diff
@@ -1578,7 +1578,11 @@ private function uploadSlicedFileToAbs(
         foreach ($slices as $slice) {
             $blobClient->createBlockBlob(
                 $prepareResult['absUploadParams']['container'],
-                basename($slice),
+                sprintf(
+                    '%s%s',
+                    $prepareResult['absUploadParams']['blobName'],
+                    basename($slice)
+                ),
```

Comment:
Tohle jsem musel upravit, aby slice byl prefixovaný stejně jako manifest. Obdobně to je u S3.
Pokud to tak nebylo tak nefungoval import, protože to při validaci validuje jestli cesta ke slicům začíná na absPath.